README.md

```diff
@@ -23,6 +23,28 @@ You will only have to run this again, if any dependencies have been changed in t
 
 To compile new target files, you will have to run: `npm run build`
 
+## Docker
+
+To build a production ready Docker image, you have to run:
+
+```sh
+docker build -t image_name:tag -f docker/images/Dockerfile .
+```
+
+or you can pull the image from the official repository:
+
+```sh
+docker pull skycoin/skycoin-web:<tag>
+```
+
+For create a container you must run:
+
+```sh
+docker run -d --name <name> -p 80:80 skycoin/skycoin-web:latest
```

Comment:
For the example I'd suggest to use a port different than `80` since that might interfere with firewalls, OS policies , ... under certain circumstances .